tests/Fixtures/path_mapping/file0

```diff
@@ -0,0 +1,2 @@
+LINE 1
```

Comment:
Might be a stupid question: Can the tests be written to use the current fixtures?

If not, that's not a big deal. But I prefer to have as little fixtures as possible, as it's hard to keep track which fixtures are used where.